.github/workflows/pull-request-check.yml

```diff
@@ -17,7 +17,7 @@ jobs:
         name: Setup Node
         uses: actions/setup-node@v2-beta
         with:
-          node-version: 12
+          node-version: 14
```

Comment:
I kept 12 everywhere because this is the only point for us where we can test that our projects will run on our servers. I no longer use this version and I don't check it against it. I always believe if something is wrong the GitHub actions will notify us.

Reply:
design and configs are probably safe to go but I wouldn't do it in other repos just yet. I am just letting you know my thinking behind upgrading things.

Reply:
Thank you for letting me know.